css/style.css

```diff
@@ -0,0 +1,128 @@
+@import url('https://fonts.googleapis.com/css2?family=Press+Start+2P&display=swap');
+
+html, body {
+   height: 100%; 
+}
+
+body {
+    background-color: #000;
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    flex-direction: column;
+}
+
+#viewport {
+    width: 640px;
+    height: 256px;
+    position: relative;
+    overflow-x: auto;
+    background-image: url("../images/water0.png");
+}
+
+#ports {
+    margin-top: 96px;
+    display: flex;
+    justify-content: space-around;
+}
+
+#ports > .port {
+    width: 64px;
+    height: 32px;
+    background-image: url("../images/port.png");
+}
+
+#ship {
+    width: 128px;
+    height: 64px;
+    background-image: url("../images/ship.png");
+    position: absolute;
+}
+
+#sailbutton {
+    border: 0;
+    background-color: #FB3640;
+    border-bottom: 4px solid #d8040f;
+    color: #FFFFFF;
+    font-family: inherit;
+    padding: 10px 20px;
+    position: absolute;
+    bottom: 0;
+    left: 0;
+    width: 100%;
+    font-family: "Press Start 2P", cursive;  
+}
```

Comment:
currently, if you add enough ports the viewport will scroll but the `sailButton` and `message` will stay in place. The easiest way to fix this would be to give them the `position: fixed`. and then set the `top`, `bottom`, `left`, `right` properties to position them where you want. Though that can be messy, an you may have to change some other properties to get it looking how you want.

There is a better way to do it, and that would be to have a `container` around your `viewport` and then have the `sailButton` and `message` be siblings of `viewport` and children of the `container`, instead of children of `viewport`. Then only allowing scroll on `viewport`. I don't want to go over too much details here and it's probably more work then it's worth, but here's a [link to my cruise-ship-gui](https://github.com/PerryBaran/cruise-ship-gui) so you can have a look at how I did it if you want to try and do it that way.